src/applications/letters/containers/AddressSection.jsx

```diff
@@ -372,14 +381,35 @@ export class AddressSection extends React.Component {
       );
     }
 
+    let viewLettersButton;
+    if (location.pathname === '/confirm-address') {
+      viewLettersButton = (
+        <div className="step-content">
+          <button
+            onClick={this.navigateToLetterList}
+            className="usa-button-primary view-letters-button"
+            disabled={
+              emptyAddress ||
+              (this.props.isEditingAddress && !forceEnableViewLettersButton)
+            }
+          >
+            View Letters
+          </button>
+        </div>
+      );
+    }
+
     return (
       <div>
-        <Element name="addressScrollElement" />
-        <div aria-live="polite" aria-relevant="additions">
-          {/* Warning message goes here while editing with no address on record */}
-          {emptyAddress && this.props.isEditingAddress && noAddressBanner}
-          {addressContent}
+        <div>
+          <Element name="addressScrollElement" />
+          <div aria-live="polite" aria-relevant="additions">
+            {/* Warning message goes here while editing with no address on record */}
+            {emptyAddress && this.props.isEditingAddress && noAddressBanner}
+            {addressContent}
+          </div>
         </div>
+        {viewLettersButton}
```

Comment:
I think you could add the `vads-u-display--block` [utility class](https://design.va.gov/utilities/display) here and avoid the need for nesting another div